web/src/styles/dashboard.css

```
@@ -0,0 +1,660 @@
+.dashboard-stat-label {
+  font-size: 0.72rem;
+  color: var(--color-text-secondary);
+  text-transform: uppercase;
+  letter-spacing: 0.16em;
+}
+
+.dashboard-stat-value {
+  font-size: 1.18rem;
+  font-weight: 700;
+  line-height: 1.15;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)